pyproject.toml

```diff
@@ -25,6 +25,7 @@ classifiers = [
 
 dependencies = [
     "importlib-metadata>=4.0",
+    "matplotlib==3.10.0",
```

Comment:
Does this version conform with the matplotlib version in FBU_Deps?

Reply:
@mkundu1 @seanpearsonuk The version in FBU_Deps is 3.6.0 which is very old. We need to update it to 3.10.0.

Reply:
If we don't want to upgrade the matplotlib version in FBU_Deps and still be compatible with PyConsole, please use "matplotlib>=3.6.0". That's what we have done in visualization.

Reply:
@prmukherj @hpohekar 

Please refer to my previous comments. For the purposes of using `matplotlib.units.ConversionInterface`, `pint` has an **optional** dependency on `matplotlib` which is not pinned to any version. The discussion about matplotlib versions should be out of scope here.

Reply:
@hpohekar Yes, inevitably, there needs to be a compatibility check in the code itself whenever the user tries to use the functionality, but there is nothing to check at the configuration level: the user can use **any** `matplotlib` version they like in their environment for whatever purpose -- independent of PyAnsys Units usage.

Reply:
I have set this PR to draft to allow @hpohekar to work on the updates without addressing further comments on code that needs to change anyway.

Reply:
got it 👍